### DISCUSSION TITLE
Windows build - Error - scrap v0.5.0

### DISCUSSION THREAD

**Discussion by ackid32** [rustdesk/rustdesk#2163] [Q&A]
failed to run custom build command for `scrap v0.5.0 (H:\rustdesk\libs\scrap)

Caused by:
  process didn't exit successfully: `H:\rustdesk\target\debug\build\scrap-62a284e00cf2fbf2\build-script-build` (exit code: 101)
  --- stdout
  cargo:info=x64-windows-static
  cargo:rustc-link-lib=static=yuv
  cargo:rustc-link-search=c:\vcpkg\installed\x64-windows-static\lib
  cargo:include=c:\vcpkg\installed\x64-windows-static\include
  cargo:info=x64-windows-static
  cargo:rustc-link-lib=static=vpx
  cargo:rustc-link-search=c:\vcpkg\installed\x64-windows-static\lib
  cargo:include=c:\vcpkg\installed\x64-windows-static\include
  rerun-if-changed=H:\rustdesk\libs\scrap\vpx_ffi.h
  rerun-if-changed=c:\vcpkg\installed\x64-windows-static\include

  --- stderr
  H:\rustdesk\libs\scrap\vpx_ffi.h:1:10: fatal error: 'vpx/vp8.h' file not found
  H:\rustdesk\libs\scrap\vpx_ffi.h:1:10: fatal error: 'vpx/vp8.h' file not found, err: true
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', libs\scrap\build.rs:134:18

How to solve this?

**Comment by sch2998** [comment]
Me too

**Comment by Goodahome** [comment]
https://rustdesk.com/docs/zh-cn/dev/build/windows/
chinese user need vpn with all steps,otherwise It's have many problems

**Comment by jzdcf** [comment]
When I use the C++v143 toolset, this error no longer occurs
